python/paddle/fluid/layers/detection.py

```diff
@@ -2221,3 +2222,58 @@ class number
     output.stop_gradient = True
 
     return output
+
+
+def collect_fpn_proposals(multi_rois,
+                          multi_scores,
+                          min_level,
+                          max_level,
+                          post_nms_topN,
+                          name=None):
+    """
+    Collect multi-level rois and select N rois with respect to multi_scores.
+    This operation performs as following steps:
+
+    1. Choose num_lvl rois and scores as input: num_lvl = max_level - min_level
```

Comment:
better to use full name num_level in doc？

Reply:
Done